### PR TITLE
MascotTailAnimation

### PR DESCRIPTION
**Pull request by Nifikus**

### CODE REVIEW

File: container/src/app/login/login.component.ts [hunk 1]
```diff
@@ -22,6 +23,17 @@ interface Locale {
...
+  // animations: [
+  //   trigger('forwards', [
+  //     animate("3s", keyframes([//барабарабара
+  //       style({ backgroundColor: "red", offset: 0 }),
+  //       style({ backgroundColor: "blue", offset: 0.2 }),
+  //       style({ backgroundColor: "orange", offset: 0.3 }),
+  //       style({ backgroundColor: "black", offset: 1 })
+  //     ]))
+  //   ])
+  // ],
```
Comment (hawkrai) [inline container/src/app/login/login.component.ts]: remove any commented code

File: container/src/app/login/login.component.ts [hunk 2]
```diff
@@ -138,4 +151,16 @@ export class LoginComponent implements OnInit {
   public open() {
     const dialogRef = this.dialog.open(VideoComponent)
   }
+
+  mascotTailAnimation(): void {
+    let catsTail = document.getElementById('mascot-tail');
```
Comment (hawkrai) [inline container/src/app/login/login.component.ts]: should be const

**Review by hawkrai — CHANGES_REQUESTED** [review]

**Review by hawkrai — CHANGES_REQUESTED** [review]
please pay attention to the comments

File: container/src/app/login/login.component.ts (hunk 2 above)
Comment (Nifikus) [reply container/src/app/login/login.component.ts]: I fixed everything as you asked. Please check it again

**Review by SergeiMol — APPROVED** [review]

**Review by hawkrai — APPROVED** [review]

**Pull request merged**